packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -85,6 +85,7 @@
   private final String cameraName;
   private final Size captureSize;
   private final Size previewSize;
+  private final boolean enableTakePictureWithMaxResolution;
```

Comment:
What is the argument for this being a boolean, rather than two independent resolution options? If at some point someone wants a different but non-max option for photos, adding that would make the API extremely messy.